packages/f-registration/src/components/Registration.vue

```diff
@@ -1,204 +1,206 @@
+                </form-field>
 
-            <form-button
-                data-test-id="create-account-submit-button"
-                button-style="primary"
-                is-full-width
-                :disabled="shouldDisableCreateAccountButton">
-                {{ buttonText }}
-            </form-button>
-        </form>
-        <p :class="$style['c-legal-hyperlinks']">
-            {{ copy.navLinks.termsAndConditions.prefix }}
-            <a
-                data-test-id="ts-and-cs-link"
-                :href="copy.navLinks.termsAndConditions.url"
-                target="_blank">{{ copy.navLinks.termsAndConditions.text }}</a>{{ copy.navLinks.termsAndConditions.suffix }}
-            {{ copy.navLinks.privacyPolicy.prefix }}
-            <a
-                data-test-id="privacy-policy-link"
-                :href="copy.navLinks.privacyPolicy.url"
-                target="_blank">{{ copy.navLinks.privacyPolicy.text }}</a>
-            {{ copy.navLinks.cookiesPolicy.prefix }}
-            <a
-                data-test-id="cookies-policy-link"
-                :href="copy.navLinks.cookiesPolicy.url"
-                target="_blank">{{ copy.navLinks.cookiesPolicy.text }}</a>{{ copy.navLinks.cookiesPolicy.suffix }}
-        </p>
-    </card>
+                <form-button
+                    data-test-id="create-account-submit-button"
+                    button-style="primary"
+                    is-full-width
+                    :disabled="shouldDisableCreateAccountButton">
+                    {{ buttonText }}
+                </form-button>
+            </form>
+            <p :class="$style['c-legal-hyperlinks']">
+                {{ copy.navLinks.termsAndConditions.prefix }}
+                <a
+                    data-test-id="ts-and-cs-link"
+                    :href="copy.navLinks.termsAndConditions.url"
+                    target="_blank">{{ copy.navLinks.termsAndConditions.text }}</a>{{ copy.navLinks.termsAndConditions.suffix }}
+                {{ copy.navLinks.privacyPolicy.prefix }}
+                <a
+                    data-test-id="privacy-policy-link"
+                    :href="copy.navLinks.privacyPolicy.url"
+                    target="_blank">{{ copy.navLinks.privacyPolicy.text }}</a>
+                {{ copy.navLinks.cookiesPolicy.prefix }}
+                <a
+                    data-test-id="cookies-policy-link"
+                    :href="copy.navLinks.cookiesPolicy.url"
+                    target="_blank">{{ copy.navLinks.cookiesPolicy.text }}</a>{{ copy.navLinks.cookiesPolicy.suffix }}
+            </p>
+        </card>
+    </div>
 </template>
 
 <script>
 import { globalisationServices } from '@justeat/f-services';
 import { validationMixin } from 'vuelidate';
-import {
-    required, email, minLength, maxLength
-} from 'vuelidate/lib/validators';
-import { WarningIcon } from '@justeat/f-vue-icons';
+import { required, email, minLength, maxLength } from 'vuelidate/lib/validators';
```

Comment:
This line is causing a lint (see the CI build logs).

```suggestion
import { 
    required,
    email,
    minLength,
    maxLength
} from 'vuelidate/lib/validators';
```